README.md

```diff
@@ -27,6 +27,8 @@ The API uses Google OAuth for authentication. You'll need to create a Project us
 #### Configuring Google OAuth
 The following steps walk you through configuring Google OAuth, with the assumption you are using it for development in conjunction with the [OneRepoToRuleThemAll](https://github.com/rit-sse/OneRepoToRuleThemAll) project. **If you are using another client, you will have to modify Step 6 to conform with that client's configuration.**
 
+Note - OAuth seems to silently fail on Windows, even running with WSL 1. It is suggested you use a unix environment (macOS or a Linux distro)
```

Comment:
Do we have an issue tracking this we can link to? Additionally, if you have Windows 10 Pro with Hyper-V, did it silently fail in Docker? If not, did you setup the environment outside of Docker? Just to assist those who try to pick it up next know what you tried.

Reply:
I didn't bother with docker or Hyper-V. Just setup wsl and tried running it natively like in a linux environment